ACActionCable.podspec

```diff
@@ -8,7 +8,7 @@
 
 Pod::Spec.new do |spec|
   spec.name                  = 'ACActionCable'
-  spec.version               = '2.0.0'
+  spec.version               = '2.1.0'
```

Comment:
Should line 16 below be updated to v2.1.0 like it was in [your first PR](https://github.com/High5Apps/ACActionCable/pull/1/files#diff-55bbb94e52e7859ea259723b0d1d3171bf7646dc1f9f80d47a915b9da3fbd8ee)?

And similar for [these places in the .pbxproj file](https://github.com/High5Apps/ACActionCable/pull/1/files#Shouldn't this be updated to v2.1.0 like it was in [your first PR](https://github.com/High5Apps/ACActionCable/pull/1/files#diff-55bbb94e52e7859ea259723b0d1d3171bf7646dc1f9f80d47a915b9da3fbd8ee)?